bin/bl_exit.py

```diff
@@ -0,0 +1,302 @@
+from bl_exit_base import BL_Exit_Base
+import ConfigParser
+import pygtk
+pygtk.require('2.0')
+import gtk
+
+class BL_Exit(BL_Exit_Base):
+		"""A dialog offering the user to log out, suspend, reboot or shut down.
+		"""
+
+		def __init__(self, cp, config_file):
+			BL_Exit_Base.__init__(self)
+			self.window = gtk.Window(gtk.WINDOW_TOPLEVEL)
+			self.window.set_name('BL Exit')
+			self.cp = cp
+			self.config_file = config_file
+			self.debug = False
+			self.selectedAction = None
+			self.window.set_decorated(True)
+			self.window.connect("delete_event", self.destroy)
+			self.window.connect("destroy_event", self.destroy)
+			# Keyboard events will need to be handled here.
+			# We need one that monitors for a focus change
+			# And changes the background to the same color
+			# As the value stored in the rc file
+			#
+			# a Listener for <-- and --> keys, also tab and shift-tab
+			# This listener will set the focus on the next or previous
+			# button in the array, we hope.
+			#
+			# An enter listener can throw an on_click
+			#
+			# We will be sure to also set focus on_click later, when
+			# Buttons are created.
+			self.window.set_resizable(False)
+			self.window.set_keep_above(True)
+			self.window.stick()
+			self.window.set_position(gtk.WIN_POS_CENTER)
+			windowicon = self.window.render_icon(gtk.STOCK_QUIT, gtk.ICON_SIZE_DIALOG)
+			self.window.set_icon(windowicon)
+
+		def configure(self):
+			if self.config_file:
+				try:
+					self.cp.read(self.config_file)
+				except ConfigParser.ParsingError as e:
+					# Log a [DEBUG} message here
+					print_message("{}: {}".format(__me__, str(e)))
```

Comment:
__me__ is out of scope.

Reply:
This sounds great. I am currently linting the code through pep8. As soon as I am done with that, I will work on these changes.

Reply:
I am pretty sure I was able to cover this one in the subsequent pr. Please refer to the code in exit_config.py in my subsequent pull request.